### PR TITLE
Add page on sending ERC-20s via XCM

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This adds a new page to the XC-20s section of the docs on how to send ERC-20s using the X-Tokens Pallet and Precompile. It also moves away from mintable xc-20s, favoring local xc-20s instead.

~~Still need to figure out the naming situation but for now just referring to regular old ERC-20s as ERC-20s... groundbreaking, I know.~~

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

**Comment by albertov19** [comment]
Hey because this is still being worked on right now, let's convert this into a draft

**Comment by eshaben** [comment]
Once other XCM pages are updated to XCM v3, I'll circle back to update the `send-erc20s.md` file to include these snippets: 'text/xcm/xcm-precompile-multilocation.md' and 'text/xc-20/xtokens/multilocation-structures.md'. Didn't want to go down the rabbit hole of updating these snippets and the pages they are on in this PR

**Comment by eshaben** [comment]
@albertov19 this can be reviewed. I only updated the new page to reflect changes to XCM v3. I've been making changes to other related pages in another branch, so I'll open another PR for those pages 🙂

File: builders/interoperability/xcm/xc20/.pages [hunk 1]
```diff
@@ -3,6 +3,5 @@ nav:
   - index.md
   - 'Overview': 'overview.md'
   - 'External XC-20s': 'xc20.md'
-  - 'Mintable XC-20s': 'mintable-xc20.md'
-  - 'Send XC-20s': 'xtokens.md'
-
+  - 'Send External XC-20s': 'xtokens.md'
+  - 'Send Local ERC-20s via XCM': 'xcm-erc20s.md'
```
Comment (eshaben) [inline builders/interoperability/xcm/xc20/.pages:7]: moved this to the bottom as it doesn't follow the same format as the 'External XC-20s' page and more aligns with the 'Send External XC-20s' page. Not sure about the name, but I did want to make it clear for those searching that this page is related to sending ERC-20s (for SEO purposes)

File: builders/interoperability/xcm/xc20/xcm-erc20s.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+
+### Use the Polkadot.js API to Send an ERC-20 Cross-Chain {: #use-polkadotjs-apps }
+
+This section of the guide will show you how to build and send an XCM message to transfer an ERC-20 cross-chain using the Polkadot.js API. This is purely for example purposes only. Before sending assets cross-chain, make sure that the asset you're sending is [registered in the target chain](#register-an-erc-20-on-another-chain); otherwise, the call will result in the loss of the assets.
+
+Since you'll be interacting with the `transfer` function of the X-Tokens Pallet, you'll need to gather the arguments for the `currencyId`, `amount`, `dest`, and `destWeight`. For this example, you can send 1 ERC-20 token to an account on the relay chain:
+
+- The `currencyId` for an ERC-20 token has the following parameters:
+
+    |    Parameter    |         Value         |
```
Comment (eshaben) [inline builders/interoperability/xcm/xc20/xcm-erc20s.md:70]: I've been thinking about this and I think tomorrow I'm going to remove these tables. It made sense to have tables like these when using Polkadot.js Apps, but now I think it just makes more sense to focus on the JavaScript code

**Comment by eshaben** [comment]
With all the changes I ended up moving the xtokens v3 update into this PR 😅

File: builders/interoperability/xcm/xc20/overview.md [hunk 3]
```diff
@@ -141,4 +276,126 @@ The **IERC20** precompile for the XC-20 will appear in the list of **Deployed Co
...
\ No newline at end of file
+To learn how to interact with each of the functions, you can check out the [ERC-20 Precompile](/builders/pallets-precompiles/precompiles/erc20/){target=_blank} guide and modify it for interacting with the XC-20 Precompile.
+
+## Register Moonbeam-based Assets on Another Chain {: #register-an-xc20 }
+
+In order to enable cross-chain transfers of Moonbeam native assets or ERC-20s between your chain and Moonbeam, you'll need to register the asset(s). To do so, you'll need the multilocation of each asset.
+
+There are additional steps aside from register assets that will need to be taken to enable a cross-chain integration with Moonbeam. For more information, please refer to the [Establishing an XC Integration with Moonbeam](/builders/interoperability/xcm/xc-integration){target=_blank} guide.
+
+### Register Moonbeam Native Tokens {: #moonbeam-native-tokens }
```
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:287]: I know we talked about having this here, but not sure it really makes sense 🤷‍♀️

**Review by albertov19 — APPROVED** [review]
LGTM! Let's doo iitttttt!

**Comment by albertov19** [comment]
@eshaben I've approved the PR, please check my commits just to check everything looks good :)

File: .snippets/text/xcm/xcm-precompile-multilocation.md [hunk 4]
```diff
@@ -39,12 +39,14 @@ Suppose the bytes array contains data. Each element's first byte (2 hexadecimal
...
-|:--------------:|:----------------------:|:-------------------------:|
-|   Parachain    |    "0x00+000007E7"     |     Parachain ID 2023     |
-|  AccountId32   | "0x01+AccountId32+00"  | AccountId32, Network Any  |
-|  AccountKey20  | "0x03+AccountKey20+00" | AccountKey20, Network Any |
-| PalletInstance |       "0x04+03"        |     Pallet Instance 3     |
+|    Selector    |       Data Value       |             Represents             |
+|:--------------:|:----------------------:|:----------------------------------:|
+|   Parachain    |    "0x00+000007E7"     |         Parachain ID 2023          |
+|  AccountId32   | "0x01+AccountId32+00"  | AccountId32, Network(Option) Null  |
+|  AccountId32   | "0x01+AccountId32+03"  |   AccountId32, Network Polkadot    |
```
Comment (eshaben) [inline .snippets/text/xcm/xcm-precompile-multilocation.md:46]:
```suggestion
|  AccountId32   | "0x01+AccountId32+02"  |   AccountId32, Network Polkadot    |
```

Polkadot should be 2

File: builders/interoperability/xcm/xc20/overview.md [hunk 5]
```diff
@@ -9,55 +9,184 @@ description:  Learn how to access and interact with an ERC-20 interface for cros
...
+XC-20s are a unique asset class on Moonbeam. It combines the power of Substrate assets (native interoperability) but allows users and developers to interact with them through a familiar [ERC-20 interface](#the-erc20-interface){target=_blank}. On the EVM side, XC-20s have [ERC-20 interface](#the-erc20-interface){target=_blank} so smart contracts and users can easily interact with them, and no knowledge of Substrate is required. This ultimately provides greater flexibility for developers when working with these types of assets and allows seamless integrations with EVM-based smart contracts such as DEXs and lending platforms, among others. Moreover, developers can integrate XC-20s with regular [Ethereum development frameworks](/builders/build/eth-api/dev-env/){target=_blank} or dApps, and create connected contracts strategies with such assets. Moreover, with the introduction of [RT2301](https://github.com/PureStake/moonbeam/tree/runtime-2301){target=_blank}, all ERC-20s are XCM-ready, meaning they can also be referred to as XC-20s.
```
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:16]:
```suggestion
XC-20s are a unique asset class on Moonbeam. It combines the power of Substrate assets (native interoperability) but allows users and developers to interact with them through a familiar [ERC-20 interface](#the-erc20-interface){target=_blank}. On the EVM side, XC-20s have an [ERC-20 interface](#the-erc20-interface){target=_blank}, so smart contracts and users can easily interact with them, and no knowledge of Substrate is required. This ultimately provides greater flexibility for developers when working with these types of assets and allows seamless integrations with EVM-based smart contracts such as DEXs and lending platforms, among others. Moreover, developers can integrate XC-20s with regular [Ethereum development frameworks](/builders/build/eth-api/dev-env/){target=_blank} or dApps, and create connected contracts strategies with such assets. Moreover, with the introduction of [RT2301](https://github.com/PureStake/moonbeam/tree/runtime-2301){target=_blank}, all ERC-20s are XCM-ready, meaning they can also be referred to as XC-20s.
```

File: builders/interoperability/xcm/xc20/overview.md [hunk 6]
```diff
@@ -9,55 +9,184 @@ description:  Learn how to access and interact with an ERC-20 interface for cros
...
+
+### Local XC-20s vs External XC-20s {: #xc-20-comparison }
+
+Both types of XC-20s can be easily sent to other parachains in the ecosystem as if they were Substrate assets, through both the Ethereum and Substrate API. However, using the Substrate API for XCM transfer will emit EVM logs for local XC-20s, but not for external XC-20s. Using the Ethereum API is recommended to provide more visiblity into the XCM actions through EVM-based explorers, such as [Moonscan](https://moonscan.io){target=_blank}.
+
+Within Moonbeam, local XC-20s can only be transferred through their regular ERC-20 interface. On the contrary, external XC-20s can be transferred through both interfaces (Substrate and ERC-20). If external XC-20s are transferred through the Substrate API, the transaction won't be visible from EVM-based block explorers. Only transactions done via the Ethereum API are visible through such explorers.
+
+
+The main difference between these two types of assets is that local XC-20s are EVM ERC-20s that have XCM-capabilities, while external XC-20s are Substrate assets with an ERC-20 interface on top.
```
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:39]:
```suggestion
The main difference between these two types of assets is that local XC-20s are EVM ERC-20s that have XCM capabilities, while external XC-20s are Substrate assets with an ERC-20 interface on top.
```

File: builders/interoperability/xcm/xc20/overview.md [hunk 7]
```diff
@@ -9,55 +9,184 @@ description:  Learn how to access and interact with an ERC-20 interface for cros
...
+
+Cross-chain transfers of XC-20s are done using the [X-Tokens Pallet](/builders/interoperability/xcm/xc20/xtokens/){target=_blank}. To learn how to use the X-Tokens Pallet to transfer XC-20s, you can refer to the [Using the X-Tokens Pallet To Send XC-20s](/builders/interoperability/xcm/xc20/xtokens){target=_blank} guide. 
+
+## Register Local XC-20s on Other Parachains {: #register-local-xc20 }
+
+In order to enable cross-chain transfers of Moonbeam local XC-20s (XCM-enabled ERC-20s) between your chain and Moonbeam, you'll need to register the asset(s). To do so, you'll need the multilocation of each asset. The multilocation will include the parachain ID of Moonbeam, the pallet instance, and the address of the ERC-20. The pallet instance will be `48`, which corresponds to the index of the ERC-20 XCM Bridge Pallet, as this is the pallet that enables any ERC-20 to be transferred via XCM.
+
+**Local XC-20s that are registered on other parachain must comply with the standard ERC-20 interface as described in the [EIP-20](https://eips.ethereum.org/EIPS/eip-20){target=_blank}.**
```
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:47]:
```suggestion
**Local XC-20s that are registered on other chains must comply with the standard ERC-20 interface as described in [EIP-20](https://eips.ethereum.org/EIPS/eip-20){target=_blank}.**
```

File: builders/interoperability/xcm/xc20/overview.md [hunk 8]
```diff
@@ -9,55 +9,184 @@ description:  Learn how to access and interact with an ERC-20 interface for cros
...
+        ]
+      }
+    }
+    ```
+
+There are additional steps aside from register assets that will need to be taken to enable a cross-chain integration with Moonbeam. For more information, please refer to the [Establishing an XC Integration with Moonbeam](/builders/interoperability/xcm/xc-integration){target=_blank} guide.
+
+## Current List of External XC-20s {: #current-xc20-assets }
+
+External XC-20s are registeredThe current list of available external XC-20 assets per network is as follows:
```
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:78]:
```suggestion
The current list of available external XC-20 assets per network is as follows:
```

File: builders/interoperability/xcm/xc20/overview.md [hunk 9]
```diff
@@ -9,55 +9,184 @@ description:  Learn how to access and interact with an ERC-20 interface for cros
...
+--8<-- 'code/xc20/retrieve-xc20s.js'
+```
 
-## XC-20s vs ERC-20 {: #xc-20-vs-erc-20 }
+The result will display the asset ID along with some additional information for all of the registered external XC-20s. 
 
-Although XC-20s and ERC-20s are very similar, some distinct differences are to be aware of. 
 
-First and foremost, XC-20s are Substrate-based assets, and as such, they are also subject to be directly impacted by Substrate features such as governance. In addition, XC-20s transactions done via the Substrate API won't be visible from EVM-based block explorers such as [Moonscan](https://moonscan.io){target=_blank}. Only transactions done via the Ethereum API are visible through such explorers.
+## XC-20s Solidity Interface
```
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:167]:
```suggestion
## XC-20s Solidity Interface {: #xc20s-solidity-interface }
```

**Comment by eshaben** [comment]
This PR can be merged once this one is approved: https://github.com/PureStake/moonbeam-docs-cn/pull/276

**Pull request merged**